scripts/npm-stats.ts

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+import { JSDOM } from 'jsdom';
+
+async function getStatsForPackage(packageName) {
+  const { data } = await axios.get(
+    `https://www.npmjs.com/package/${packageName}?activeTab=versions`
+  );
+
+  const dom = new JSDOM(data);
+  const ul = dom.window.document.querySelector(
+    '.a0dff0b1 ~ .c495d29d ~ .a0dff0b1 ~ .c495d29d'
+  );
+  const versionLines = ul.children;
+  const res: any[] = [];
+
+  for (let i = 1; i < versionLines.length; i++) {
+    const line = versionLines.item(i);
+    const [version, _, downloads] = line.children;
+    res.push({
+      version: version.textContent,
+      downloads: parseInt(downloads.textContent.replace(/,/g, ''))
+    });
+  }
+
+  return res.reduce((majorVersions, { version, downloads }) => {
+    if (version.includes('-')) {
+      return majorVersions;
+    }
+    const [majorVersion] = version.split('.');
+    if (majorVersions[majorVersion] === undefined) {
+      majorVersions[majorVersion] = 0;
+    }
+    majorVersions[majorVersion] += downloads;
+    return majorVersions;
+  }, {});
+}
+
+async function getAllStats() {
+  return ['@sap-cloud-sdk/util', '@sap/cds'].reduce(
+    async (stats, packageName) => ({
+      ...(await stats),
+      [packageName]: await getStatsForPackage(packageName)
+    }),
+    {}
+  );
+}
+
+async function main() {
+  // eslint-disable-next-line no-console
+  console.log(await getAllStats());
```

Comment:
Could we create a GH repo where we upload the stats every day? Maybe using this: https://github.com/octokit/octokit.js